src/main/java/uk/ac/ebi/ampt2d/metadata/persistence/repositories/TaxonomyRepository.java

```diff
@@ -18,9 +18,34 @@
 package uk.ac.ebi.ampt2d.metadata.persistence.repositories;
 
 import org.springframework.data.repository.CrudRepository;
+import org.springframework.data.repository.query.Param;
 import org.springframework.data.rest.core.annotation.RepositoryRestResource;
+import org.springframework.data.rest.core.annotation.RestResource;
 import uk.ac.ebi.ampt2d.metadata.persistence.entities.Taxonomy;
 
+import java.util.List;
+
 @RepositoryRestResource
 public interface TaxonomyRepository extends CrudRepository<Taxonomy, Long> {
+
```

Comment:
we can use QueryDsl predicates to search instead CrudRepo methods so that it looks cleaner and we don't need to add endpoints in taxonomy repo

Reply:
I have removed those endpoints and now repositories are cleaner.